### PR TITLE
Server channel redirection command

### PR DESCRIPTION
**Pull request by SpecialFlocon**

### CODE REVIEW

File: model/command/redirect.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        this.isAllowedForContext = CommandPermission.notInWelcome;
+    }
+
+    /**
+     * @param {Message} message
+     */
+    async process(message, args) {
+        if (this.canCall()) {
+            this.registerCall();
+            if (args.length != 1) {
```
Comment (LilyWonhalf) [inline model/command/redirect.js:28]: < 1

File: model/command/redirect.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                message.channel.send(trans('model.command.redirect.toClassrooms', [Guild.mainClassroomChannel.toString()]));
+
+                return;
+            }
+            if (!['chat', 'class'].includes(args[0])) {
```
Comment (LilyWonhalf) [inline model/command/redirect.js:33]: empty line before a block
conditions, loops, functions, classes are blocks

Comment (LilyWonhalf) [reply model/command/redirect.js:33]: Maybe add an alias system, let's not repeat the mistake we made with the .level command and expect the users to type exactly what we expect them to type :p .

First, `args[0]` should be `args.join(' ').toLowerCase()`
Second, you should make an object *before* the declaration of the command class, something like that:

```js
const destinations = {
    'class': 'class',
    'classe': 'class',
    'salle de classe': 'class',
    'salledeclasse': 'class',
    'classroom': 'class',
    'chat': 'chat',
    'francais': 'chat',
    'français': 'chat',
    'anglais': 'chat',
    'discussion': 'chat',
    'conversation': 'chat'
}
```

That way, you can use `Object.keys(destinations).includes(args.join(' ').toLowerCase())` for your condition, and `destinations[args.join(' ').toLowerCase()]` to retrieve the actual destination.

Might be better to store `args.join(' ').toLowerCase()` in a const.

File: model/command/redirect.js (hunk 1 above)
Comment (SpecialFlocon) [reply model/command/redirect.js]: 5c9c84e

File: model/command/redirect.js (hunk 2 above)
Comment (SpecialFlocon) [reply model/command/redirect.js]: 5c9c84e

**Review by LilyWonhalf — APPROVED** [review]

**Pull request merged**